src/data/events.yml

```diff
@@ -986,3 +986,34 @@
     more_links:
         -   title: Všechny přednášky v&nbsp;playlistu
             url: https://www.youtube.com/watch?v=qE2QWIxWUv8&list=PLnXfazh66kVddq-0dEYgvD7JD-EIklEAF
+
+-   id: 30
+    slug: chrome-dev-tools-metriky-rychlosti-nacitani-jamstack
+    title: Chrome DevTools, metriky rychlosti načítání a&nbsp;JAMstack
+    description: >
+        Středa 14.&nbsp;3. 2018, Ataccama office Praha
+    speakers:
+        -   name: Iakov Markov
+            name_link: https://twitter.com/IakovMarkov
+            company: Ataccama
+            company_link: https://www.ataccama.com/
+            title: Chrome&nbsp;DevTools, základní i&nbsp;pokročilé měření výkonu
+            youtube_id: 8DCQAJWwhFE
+            image: iakov-markov.jpg
+        -   name: Martin Michálek
+            name_link: https://twitter.com/machal
+            company: Freelancer
+            company_link: ~
+            title: Metriky rychlosti načítání a&nbsp;tipy pro WebpageTest.org
+            youtube_id: fR9n_yi050g
+            image: martin-michalek.jpg
```

Comment:
Could you please add links to Slideshare presentations of Martin and Láďa (see #27)? We don't have Iakov yet but we can add his slides later.